app/models/enrollment.rb

```diff
@@ -0,0 +1,4 @@
+class Enrollment < ApplicationRecord
```

Comment:
I liked the way he did it, `Enrollment` and a flag to indicate if one participated or not. Because you have a list with everyone who is going to participate of the event right?